clang/include/clang/Basic/DiagnosticSemaKinds.td

```diff
@@ -11794,6 +11794,9 @@ def err_ivdep_declrefexpr_arg : Error<
 def warn_ivdep_redundant : Warning <"ignoring redundant Intel FPGA loop "
   "attribute 'ivdep': safelen %select{INF|%1}0 >= safelen %select{INF|%3}2">,
   InGroup<IgnoredAttributes>;
+def warn_ivdep_attribute_argument : Warning<
+  "'ivdep' attribute with value %0 has no effect; attribute ignored">,
+  InGroup<IgnoredAttributes>;
```

Comment:
Thanks @AaronBallman for the suggestion. I have created a new group and pit it under ``IgnoredAttributes`` so that that ivdep attribute warning with value .0 and 1 will get suppressed by -Wno-ignored-attributes